### ISSUE TITLE
Superscript for ~..~ in names

### ISSUE THREAD

**Issue by ghutchis** [pittquantum/backend#42]
Some names, e.g. http://pqr.pitt.edu/mol/ACTNIVURBGOQCE-UHFFFAOYSA-N

4-methoxy-2,15-dioxapentacyclo[20.2.2.2~16,19~.1~3,7~.1~10,14~]triaconta-1(24),3(30),4,6,10(29),11,13,16,18,22,25,22-dodecahen-17-ol

This should be rendered with superscripts, e.g.
4-methoxy-2,15-dioxapentacyclo[20.2.2.2<sup>16,19</sup>.1<sup>3,7</sup>.1<sup>10,14</sup>]triaconta-1(24),3(30),4,6,10(29),11,13,16,18,22,25,22-dodecahen-17-ol

In other words, if a "name" or "iupac_name" field has ~ characters, then anything inside ~[...]~ should be transformed to <sup>[..]</sup> to render as superscripts.

**Comment by JoshuaRogan** [comment]
Can you confirm this is correct? 

It works for the one you suggested and should work for them all but I couldn't find any more to test.
http://pqr.melwood.jcubedworld.com/mol/ACTNIVURBGOQCE-UHFFFAOYSA-N/

**Comment by RitwikGupta** [comment]
It looks right to me

**Comment by RitwikGupta** [comment]
Also make that entire thing lowercase

**Comment by JoshuaRogan** [comment]
Both are done

**Comment by ghutchis** [comment]
Awesome! Great to squash something quick. :-)
